mule-packager/pom.xml

```diff
@@ -51,14 +51,17 @@
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-core</artifactId>
+            <scope>test</scope>
```

Comment:
please remove the scope as they're being taken from the parent and are test already